main.js

```diff
@@ -1,3 +1,4 @@
+// portfolio-mobile menu
```

Comment:
As the project requires the mobile menu should appear and disappear when the user clicks the hamburger button or when chooses an option from the menus. Kindly, fix the scrollable behavior of the mobile menu it should be fixed. You may use position `fixed` only for the `pop-up` class.

## Refer to the video below 👇 

https://user-images.githubusercontent.com/34578631/162697222-f0177c45-3d72-4783-a815-a70477575e0d.mov

To stick to the design you can use background color something like this `background: #1127e5f2;` or you can use background `filter: blur(...)`.